cli_045_gcc_co2_concentrations/time.cron

```diff
@@ -0,0 +1 @@
+50 2 * * WED
```

Comment:
Since this dataset updates monthly and I'm not really sure what day it updates, I'm gonna go ahead and run the updates weekly to make sure we don't end up being a month behind the source with updates. I see that the dates are usually around the middle of the month, but June data hasn't been added yet. If you wanted, you could monitor the source to see when it updates with June data, then check it the next couple months to see if it updates on the same day each month. Then choose the a date that you are pretty confident is right after the source update (so we don't miss the update). But if you don't want to do all that, this script probably doesn't take long to run, so running it weekly should be fine.